### PR TITLE
ioctl action transfer uses IOTX for amount rather than Rau #1215

### PR DESCRIPTION
**Pull request by lzxm160**
work on #1215 

1、ioctl account balance bb
io1mflp9m6hcgm2qcghchsdqj3z3eccrnekx9p0ms: 0 IOTX

2、ioctl action transfer bb 1.111111111 -s aa
Enter password #aa:

version: 1  nonce: 4  gasLimit: 10000  gasPrice: 1000000000000 Rau
senderAddress: io1vdtfpzkwpyngzvx7u2mauepnzja7kd5rryp0sg (aa)
transfer: <
  recipient: io1mflp9m6hcgm2qcghchsdqj3z3eccrnekx9p0ms (bb)
  **amount: 1.111111111 IOTX**
>
senderPubKey: 04ea8046cf8dc5bc9cda5f2e83e5d2d61932ad7e0e402b4f4cb65b58e9618891f54cba5cfcda873351ad9da1f5a819f54bba9e8343f2edd1ad34dcf7f35de552f3
signature: ba9822f15390ac48c63bc0efb9c2decfcc765f347ca3a92ed6892c2291f14f674d8b0f2e4a410ea88211e6b164d6d1cfcbf3a4d47a23f48786d3a678e730f1d801

Please confirm your action.
Type 'YES' to continue, quit for anything else.
yes

Action has been sent to blockchain.
Wait for several seconds and query this action by hash:
87109576e6335e1154b7a12d5c1855c0e52c0f81ac2ad0179854a5cb70b4728d

3、ioctl account balance bb
io1mflp9m6hcgm2qcghchsdqj3z3eccrnekx9p0ms: 1.111111111 IOTX

### CODE REVIEW

File: cli/ioctl/cmd/action/actionhash.go [hunk 1]
```diff
@@ -114,7 +114,15 @@ func printAction(actionInfo *iotexapi.ActionInfo) (string, error) {
 	output += fmt.Sprintf("actHash: %s\n", actionInfo.ActHash)
 	return output, nil
 }
-
+func toIOTX(amount string) (iotx string, err error) {
+	amountInt, err := util.StringToRau(amount, 0)
```
Comment (raullenchai) [inline cli/ioctl/cmd/action/actionhash.go]: This is confusing. Can you please implement StringToIOTX() in util/util.go?

File: cli/ioctl/cmd/action/actionhash.go [hunk 2]
```diff
@@ -114,7 +114,15 @@ func printAction(actionInfo *iotexapi.ActionInfo) (string, error) {
...
+	if err != nil {
+		log.L().Error("failed to convert amount into int", zap.Error(err))
+		return "", err
+	}
+	iotx = util.RauToString(amountInt, 18)
+	return
```
Comment (raullenchai) [inline cli/ioctl/cmd/action/actionhash.go]: `return` not needed

File: cli/ioctl/cmd/action/actionhash.go (hunk 1 above)
Comment (lzxm160) [reply cli/ioctl/cmd/action/actionhash.go]: sure

File: cli/ioctl/cmd/action/actionhash.go (hunk 2 above)
Comment (lzxm160) [reply cli/ioctl/cmd/action/actionhash.go]: build needed

**Pull request merged**